### DISCUSSION TITLE
can i make an os like windows with this

### DISCUSSION THREAD

**Discussion by Hasnain12340** [CosmosOS/Cosmos#2484] [Q&A]
can i?

**Comment by selkij** [comment]
Nope windows os really is sofisticated and even with the help of cosmos it would take atleast 10years

**Reply by Hasnain12340** [reply]
Ok, so what are the key things you can do with cosmos

**Comment by selkij** [comment]
Cosmos is a kid playground compared go windows

**Reply by Hasnain12340** [reply]
oh lol ok

**Comment by terminal-cs** [comment]
you _can_, but it will take roughly the same amount of effort it took to make windows, plus all the extra bugs if you don't make your own standard library
